BibBuddy/app/src/main/java/de/bibbuddy/LibraryRenameShelfFragment.java

```diff
@@ -38,6 +38,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     setupButtons(view, bundle);
     setupEditText(view, bundle.getString(LibraryKeys.SHELF_NAME));
 
+    ((MainActivity) getActivity()).setVisibleImportShareButton(View.INVISIBLE, View.INVISIBLE);
```

Comment:
I would set the buttons to invisible by default in the xml and then show them when needed.

Reply:
See the conversation above.